lib/mongo-stream.js

```diff
@@ -2,9 +2,11 @@
 
 var WritableStream = require('stream').Writable
   , util = require('util')
-  , dao = require('./dao/bunyan-logs');
+  , dao = require('./dao/bunyan-logs'),
+  flag = false,
```

Comment:
I'm nitpicking, but could the existing style be used, or change the existing style of commas entirely 👍